make.ps1

```diff
@@ -18,17 +18,74 @@ function FindMSBuild
 	return $null
 }
 
+function CopyDependencies
+{
+	cd thirdparty
+	./fetch-thirdparty-deps.ps1
+	cp *.dll ..
+	cp windows/*.dll ..
+	cd ..
+	echo "Dependencies copied."
+
+	return $LASTEXITCODE
+}
+
+function TestModsAndMaps
+{
+	if(Test-Path ./OpenRA.Utility.exe)
+	{
+		echo "Testing mods..."
+		if(Test-Path ./mods/ra/OpenRA.Mods.RA.dll)
+		{
+			echo ""
+			echo "Testing Red Alert mod MiniYAML..."
+			./OpenRA.Utility.exe ra --check-yaml
+	
+			if(Test-Path ./mods/cnc/OpenRA.Mods.Cnc.dll)
```

Comment:
All these new checks add a bunch of complexity to the script with no real benefit.